### ISSUE TITLE
Simplify CSS for font-weight property.

### ISSUE THREAD

**Issue by josephtkim** [JiddieBoom/prj-rev-bwfs-dasmoto#7]
Great job simplifying the CSS code for the font-family property by applying it to the body selector, and for applying the properties to the h2 selector. You can also simplify the CSS for the font-weight property, which many selectors seem to share as well, possibly adding it to the body selector.

**Comment by JiddieBoom** [comment]
If I add the font-weight: bold to my body selector then my paragraphs are also bold which I do not want. There must be something I am missing?

**Comment by josephtkim** [comment]
You're right that the paragraphs will also become bold. If you did apply it to the body selector, you would then be able to set the paragraphs to normal font-weight like

```css
p {
  font-weight: normal;
}
```

below the body selector to override it. 

This might not seem too efficient, as it ends up adding code to reset the p font-weight, but in the case of maybe 10 elements that you wanted to bold and a few elements that should not be bold, it might be more efficient to select all elements first using body, and then resetting the font-weights for the few elements that should not be bold.

However, since there are a few selectors for this project, you can list those to apply the font-weight bold to specifically. Feel free to use whichever method you'd like!
